packages/agoric-cli/src/install.js

```diff
@@ -226,15 +226,14 @@ export default async function installMain(progname, rawArgs, powers, opts) {
       }
     }
   } else {
+    const subdirMapper = async subd => {
+      const exists = await fs.stat(`${subd}/package.json`).catch(_ => false);
+      return exists && subd;
+    };
```

Comment:
For all of these, someone who better understands the purpose of the code should suggest better names.

Reply:
```suggestion
    const subdirPackageJsonExists = async subd => {
      const exists = await fs.stat(`${subd}/package.json`).catch(_ => false);
      return exists && subd;
    };
```

Reply:
Done